go/README.md

```diff
@@ -28,6 +28,23 @@ for flat and hierarchical data, organized for efficient analytic operations on
 modern hardware. It also provides computational libraries and zero-copy
 streaming messaging and inter-process communication.
 
+### A note about FlightSQL drivers
+
+Go FlightSQL drivers live in the
+[ADBC repository](https://github.com/apache/arrow-adbc/tree/main/go/adbc).
+In particular, to use the Golang `database/sql` interface:
```

Comment:
we should probably document that you can pass multiple options by separating them with `;` and then point to the ADBC documentation (https://arrow.apache.org/adbc/main/driver/go/flight_sql.html) which contains the available options which can be passed in addition to the uri. such as the user and password, etc.

Reply:
Great idea.